### DISCUSSION TITLE
Stat: typing speed against time of day

### DISCUSSION THREAD

**Discussion by Thlr** [monkeytypegame/monkeytype#1951] [Ideas]
Hi team,

I like to use monkeytype first thing in the morning, and also some time in the afternoon when I get bored. I noticed, of course, that  my typing speeds are radically different between these two moments of the day.

It'd be awesome to have some stat or maybe just a filter to be able to compare typing speeds across different hours of the day, or to compare typing speeds across days at similar hours of the day. I think it would make a lot of sense to me, much more than having my typing speed averaged throughout the entire day because I don't consistently type every morning or every afternoon.

Just a quick thought, thanks for your time this website is a treat to use.

**Comment by frogfrogfrogfrogfrog** [comment]
I'm pretty sure that this solution exists. Whenever you finish a typing test the test will be recorded with the exact time of day as well. Using this information you could simply type in the morning. Type in the afternoon then, you could compare the two typing speeds by hovering over them. You can also compare the times over the course of days by using the "all time" option and by hovering over the days that you typed in the morning. If "all time" becomes too clunky you can use smaller time sets such as "last month" or "last week".

**Comment by Miodec** [comment]
Maybe you could try using tags for this? Make a `morning` and `evening` tag?
